include/oneapi/mkl.hpp

```diff
@@ -31,6 +31,6 @@
 #include "oneapi/math/rng.hpp"
 #include "oneapi/math/sparse_blas.hpp"
 
-#include "namespace_alias.hpp"
+#include "mkl/namespace_alias.hpp"
```

Comment:
Do we also need `oneapi` here, i.e., `oneapi/mkl/namespace_alias.hpp`? I didn't test it, just a quick check.

Reply:
Hi @dnhsieh-intel, I tested both fixes; they work. `oneapi/mkl/namespace_alias.hpp` is relative to a path in `CPATH`, while `mkl/namespace_alias.hpp` is relative to the current file.